cmd/contour/serve.go

```diff
@@ -639,6 +639,10 @@ func (s *Server) doServe() error {
 		s.log.WithError(err).WithField("resource", "secrets").Fatal("failed to create informer")
 	}
 
+	if err := s.informOnResource(&corev1.ConfigMap{}, handler); err != nil {
```

Comment:
i think we could only start this informer if the backendtlspolicy feature is enabled, could augment the `features` map below to be `map[string][]string` so if a feature is disabled/enabled we can enable/disable any other required informers/controllers

Reply:
Not sure I follow the change for features to go from `map[string]struct{}` to `map[string][]string`. What would go into the new string array value type here? I see the serve.go currently deletes from the map if a feature is disabled. Is that not enough?

Reply:
the map key would be feature names, which currently are only resource kinds we inform on, and then the string list could be kinds that a particular feature/top level kind depends on, in this case backendtlspolicy also needs a configmap informer to work fully, so if you disable backendtlspolicy you dont need to have a configmap informer etc.

it might be overkill, tbh since we also have an if statement checking if backendtlspolicy is enabled, we can add the creation of the configmap informer in there

Reply:
Ended up moving the informOnResource line down into the feature check for backendltspolicy.